### ISSUE TITLE
Cannot find name 'IAdvancedFilterCondition'

### ISSUE THREAD

**Issue by nrandell** [Microsoft/powerbi-visuals-utils-interactivityutils#23]
After updating to the latest version  (3.2.0) of this project, I can't build my visual. 

```
 error  TYPESCRIPT  C:/Users/nick/Source/PowerBI/scheduleVariance/node_modules/powerbi-visuals-utils-interactivityutils/lib/index.d.ts : (81,41) Cannot find name 'IAdvancedFilterCondition'.
```

However installing powerbi-models and adding the following line to files in tsconfig.json allows it to build ok.
```
    "node_modules/powerbi-models/dist/models-noexports.d.ts",
```
I think this is just a documentation update

**Comment by uve** [comment]
@nrandell good catch, we'll update the documentation.

**Comment by ignatvilesov** [comment]
Fixed in #28
